storm-client/src/jvm/org/apache/storm/task/IOutputCollector.java

```diff
@@ -30,4 +31,5 @@
     void ack(Tuple input);
     void fail(Tuple input);
     void resetTimeout(Tuple input);
+    void flush();
```

Comment:
thanks for that sweet tip! just to be sure, i would feel comfortable verifying with profiler first ... given this is in critical path.   

ok Satish let me see about that.